controller.js

```diff
@@ -0,0 +1,129 @@
+const testsList = document.getElementById("testsList");
+
+
+
+
+document.getElementById("createTestButton").querySelector("img").addEventListener("click", () => createTest() );
+document.getElementById("sendDataButton").addEventListener("click", function () {
+  sendData();
+});
+
+
+
+function sendData() {
```

Comment:
Flask backend with SocketIO wrapping guarantees that this entire function will be unused in the way it currently is.

Please either make function body empty/stud, or add in socketIO script here.

Reply:
It could be used as a placeholder and be rewritten as a SocketIO script when that part will be available.